etc/make_release.py

```diff
@@ -363,9 +367,9 @@ def build_c_driver(c_driver_install_dir, c_driver_build_ref, quiet):
 
     env = os.environ
```

Comment:
`os.environ` should be copied unless it is intended to modify the process's own environment (Writing to `env` will call `putenv()`)